iptables/iptables.go

```diff
@@ -70,6 +75,12 @@ func ConfigureFirewall(firewallConfiguration FirewallConfiguration) error {
 
 	startSection("configuration")
 
+	matches := chainRegex.FindAllString(b.String(), -1)
+	if len(matches) > 0 {
+		log.Println("Found existing firewall configuration; Skipping")
+		return nil
+	}
```

Comment:
So passing `-1` into `FindAllString` means we'll search the entire string for all possible matches. I would either print the matches as part of the `Println`, or just pass in `1` so that we stop at the first. There isn't really a reason to search the whole string if we're just going to check `len(matches) > 0`.